pkg/utils/utils.go

```diff
@@ -212,3 +212,18 @@ func DeDuplicate(input []string) []string {
 	}
 	return result
 }
+
+// StringInSlice checks if a string in the slice.
+func StringInSlice(str string, input []string) bool {
```

Comment:
Can you change this to base function, such as slice.Contains?
```golang
func Contains(input []interface{}, s interface{}) bool {
    ....
}
```

Reply:
Greate Idea, i will do it.